pep-0696.rst

```diff
@@ -190,38 +196,52 @@ default to the type of ``start`` and step default to ``int | None``.
 
    class slice(Generic[StartT, StopT, StepT]): ...
 
+   reveal_type(slice)  # type is type[slice[StartT = int, StopT = StartT, StepT = int | None]]
    reveal_type(slice())                        # type is slice[int, int, int | None]
    reveal_type(slice[str]())                   # type is slice[str, str, int | None]
    reveal_type(slice[str, bool, timedelta]())  # type is slice[str, bool, timedelta]
 
+   T2 = TypeVar("T2", default=DefaultStrT)
+
+   class Foo(Generic[DefaultStrT, T2]):
```

Comment:
This references DefaultStrT, which isn't defined in this sample.

Reply:
I was just using those from before as you might have guessed, I wasn't particularly aware of any stipulations that the snippets were meant to all contain the context to run them in an isolated way